service/controller/app/controllercontext/context.go

```diff
@@ -27,10 +27,16 @@ type Clients struct {
 }
 
 type Status struct {
-	TenantCluster TenantCluster
+	ChartStatus   ChartStatus
```

Comment:
Since we need to have status from a chart too, I think extending `Status` struct feels right.